samples/highcharts/yaxis/stackshadow/demo.html

```diff
@@ -0,0 +1,15 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
```

Comment:
This sample deserves a place in the demo page I think. Should we call it `highcharts/demo/pictorial-multiple`? The main difference as I see it is that this one has multiple stacks compared to the light bulb demo. Open for suggestions.

After moving, remember to update the reference in the doclets `@sample` tags.

Reply:
I agree, so I moved it to the main demos but I renamed it to pictorial-stackshadow - due to the main goal of this demo - showing `stackshadow` option. What do you think?